src/pages/index.tsx

```diff
@@ -167,6 +167,24 @@ const BlockList: React.FC<BlockListProps> = ({ blocks }) => {
       data: 'string',
       isPadded: true,
     },
+    {
+      key: 'columnTimeTaken',
+      name: 'Time Taken',
+      minWidth: 80,
+      maxWidth: 200,
```

Comment:
Don't other columns need to adjust their width?

Reply:
Originally *Hash* column had somewhat wide width than enough but it's automatically reduced by *Time Taken* column added. 
Also *Time Taken* has too wide width, so I reduced its width.